def bad_request(details):
    return {
        "error": "An unexpected error has occurred",
        "details" : details
    }